src/block_cache/block_cache.c

```diff
@@ -528,7 +529,15 @@ __wt_blkcache_remove(WT_SESSION_IMPL *session, const uint8_t *addr, size_t addr_
             WT_STAT_CONN_INCR(session, block_cache_blocks_removed);
             (void)__wt_atomic_sub64(&blkcache->bytes_used, blkcache_item->data_size);
             blkcache->removals++;
```

Comment:
Just a small thing. I think we should increase the counter only after we have actually freed the block.

Reply:
Also for the bytes_used

Reply:
Updated in 7bf66a5db552adc6aece0a8b556c53ded1acbf90